doc/source/actions.rst

```diff
@@ -254,6 +254,17 @@ The ``merge`` action merges the pull request into its base branch. The
        * GPG signed commits will lost their signatures.
        * Also see: :ref:`faq strict rebase`
 
+When the Pull Request is merged, the default merge/squash commit message is the
```

Comment:
When a pull request is merged using the squash or merge method, Mergify uses the default commit message provided by GitHub. You can override the commit message by adding a section in the pull request body. The section must start with the Markdown title "Commit Message" and contain the actual commit message. For example::